modules/keyboard.js

```diff
@@ -44,6 +44,7 @@ class Keyboard extends Module {
       if (handlers.length > 0) {
         let range = this.quill.getSelection();
         handlers.forEach((handler) => {
+          if (evt.defaultPrevented) return;
```

Comment:
The problem I'm trying to prevent is when the first handler in `handlers` calls evt.preventDefault(), other handlers need to not fire (specifically, the keyboard.onEnter and onTab handlers here shouldn't fire when I'm intercepting the enter / tab events with my own autocomplete code. 

I can't guarantee my own keyboard listener will fire and prevent before quill's keydown listener fires (at least, I don't think I can - I don't think eventListeners fire in a guaranteed order unless you're using the jquery flavor).